src/main/webapp/scripts/session-script.js

```diff
@@ -1,46 +1,84 @@
-// Copyright 2019 Google LLC
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     https://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
+/**
+ * An array of who is currently in the session.
+ * @type {Object}
+ */
+let currentAttendees = [];
 
 /**
- * function openSessionInfo() displays the div container
- * that has information about the session.
+ * Represents the current session, a Session object.
+ * @type {Object}
  */
-function openSessionInfo() {
-  document.getElementById('session-info-div').style.display = 'block'; 
-}
+let session;
+
+/**
+ * Represents (in miliseconds) how long users are alerted of any membership
+ * changes in the session. 
+ * @type {number}
+ */
+const DISPLAY_CADENCE = 4000;
 
 /**
- * function closeSessionInfo() closes the div container
- * that has information about the session.
+ * function updateSessionInfoAttendees() adds new attendees to the
+ * session to the session info attendee div. Also removes attendees 
+ * if they left the session. Alerts users of anyone who has left/entered.
  */
-function closeSessionInfo() {
-  document.getElementById('session-info-div').style.display = 'none';
+function updateSessionInfoAttendees() {
+  const /** Object */ updatedAttendees = session.getListOfAttendees();
+  const /** Object */ newAttendees = [];
+  const /** Object */ attendeesThatHaveLeft = [];
+  for (const attendee of updatedAttendees) {
+    if (!currentAttendees.includes(attendee)) {
+      buildAttendeeDiv(attendee)
+      newAttendees.push(attendee);
+    }
+  }
+  for (const attendee of currentAttendees) {
+    if (!updatedAttendees.includes(attendee)) {
+      removeAttendeeDiv(attendee);
+      attendeesThatHaveLeft.push(attendee);
+    }
+  }
+  if (newAttendees.length) {
```

Comment:
Please prefer "blah.length === 0" in order to check for emptiness. Dont rely on the "truthy" and "falsy" nature of non-boolean types